### DISCUSSION TITLE
PyCharm reconsider?

### DISCUSSION THREAD

**Discussion by pdjames** [kdbinsidebrains/plugin#68] [General]
Hi, 

Great work with this plugin, I've used it for a few weeks now and find it a great drop in replacement for my previous q ~ide~ (pseudo IDEs really) experiences. And some added features that are obvious improvements.

I was wondering, as I can see the pycharm issue makes the clear case for not pursuing pycharm release, since that issue is a couple of years old now if it was possible to / worth it to take another look at releasing through pycharm also?

At the moment I have some work for which I use IntelliJ only for the use of this plugin alongside entirely pycharm code and ci/cd. So it would be a 'nice to have' if I could contain the q work in the same place.

Not critical, and I see the reasons for the current state, so not urgent. But if/when possible I'd be among those to jump across to using this in pycharm. :)

Thank you for such a great plugin!

**Comment by smklimenko** [comment]
sorry, but PyCharm doesn't support some language features that I use in the plugin so it won't be supported, I'm afraid. Even when I need Python in my projects, Python for IntelliJ plugin covers all PyCharm features: https://plugins.jetbrains.com/plugin/631-python

**Comment by smklimenko** [comment]
PyCharm supporting as well as other JetBrains product has been added since v5.0.0
